### PR TITLE
More information describing a signalfx endpoint

### PR DESCRIPTION
**Pull request by harnitsignalfx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,8 +125,12 @@ When configuring CollectD, the target URL path will be `/post-collectd`.
 
 #### signalfx
 
-You can pretend to be a SignalFx endpoint with the `signalfx` type.  For this,
-you will need to specify which port to bind to.  An example config:
+You can pretend to be a SignalFx endpoint with the `signalfx` type. A SignalFx
```
Comment (mdubbyap) [inline README.md]: I wouldn't use the term "pretend to be..." i would use "You can expose a SignalFx endpoint..."

**Comment by mpetazzoni** [comment]
@harnitsignalfx since @mdubbyap is away I can take over the review of this. Let me know when you've updated the wording as per @mdubbyap's comments.

**Comment by harnitsignalfx** [comment]
@mpetazzoni I updated the verbage. Not sure why CI failed, I'm guessing it is ok to ignore. Let me know, otherwise I can delete this PR and create a new one.

**Comment by harnitsignalfx** [comment]
@mpetazzoni Hi Max, wanted to check if this can be merged? or would you want me to create a new PR?

**Comment by mpetazzoni** [comment]
Nope you can't ignore the CI failures, we validate the Markdown. You have a trailing whitespace line 131. While you're in there, make sure to finish that sentence with a full stop, and write protobuf as "Protocol Buffers" and json as "JSON".

**Pull request merged**